src/QMCWaveFunctions/Fermion/DiracDeterminantBatched.cpp

```diff
@@ -107,7 +107,7 @@ void DiracDeterminantBatched<DET_ENGINE_TYPE>::mw_evalGrad(const RefVector<WaveF
   for (int iw = 0; iw < nw; iw++)
   {
     auto& det = static_cast<DiracDeterminantBatched<DET_ENGINE_TYPE>&>(WFC_list[iw].get());
-    invRow_list[iw] = det.psiMinv_dev_ptr + NumOrbitals * WorkingIndex;
+    invRow_list[iw] = det.psiMinv_dev_ptr + psiMinv.cols() * WorkingIndex;
```

Comment:
Sometimes you use NumOrbitals sometimes a dimension of the data object that is over NumOrbitals.  If mixed naming is purposeful make a explanatory comment explaining the deviation (for instance psiMinv.cols() might be distributed or a subset?

Reply:
psiMiv.cols() is the stride. Need to revisit whether padding the NumOrbitals is necessary or not.